### PR TITLE
update circuitbreaker

### PR DESCRIPTION
**Pull request by suchen-sci**

### CODE REVIEW

File: doc/cookbook/resilience.md [hunk 1]
```diff
@@ -3,12 +3,12 @@
 - [Resilience](#resilience)
   - [Basic: Load Balance](#basic-load-balance)
   - [More Livingness: Resilience of Service](#more-livingness-resilience-of-service)
-    - [Circuit Break](#circuit-break)
+    - [CircuitBreaker](#circuitbreaker)
     - [RateLimiter](#ratelimiter)
     - [Retry](#retry)
     - [Time Limit](#time-limit)
```
Comment (xxx7xxxx) [inline doc/cookbook/resilience.md:9]: How about change `Time Limit` to `TimeLimiter` by the way

Comment (suchen-sci) [reply doc/cookbook/resilience.md:9]: There is no `TimeLimiter` in easegress 2.0. We use a simple timeout parameter in proxy.

**Comment by localvar** [comment]
please also rename `circuitbreak.go` to `circuitbreaker.go`.

File: doc/cookbook/resilience.md (hunk 1 above)
Comment (suchen-sci) [reply doc/cookbook/resilience.md]: updated!

File: pkg/filters/proxy/pool.go [hunk 2]
```diff
@@ -170,31 +170,31 @@ type ServerPool struct {
...
-	filter              RequestMatcher
-	loadBalancer        atomic.Value
-	timeout             time.Duration
-	retryWrapper        resilience.Wrapper
-	circuitbreakWrapper resilience.Wrapper
+	filter                RequestMatcher
+	loadBalancer          atomic.Value
+	timeout               time.Duration
+	retryWrapper          resilience.Wrapper
+	circuitbreakerWrapper resilience.Wrapper
```
[the remark this answers is not in the document]
Comment (suchen-sci) [reply pkg/filters/proxy/pool.go]: udpated!

**Review by localvar — APPROVED** [review]

**Comment by xxx7xxxx** [comment]
Please fix test failure

**Review by xxx7xxxx — APPROVED** [review]

**Pull request merged**